### PR TITLE
Dockerize app

### PR DESCRIPTION
**Pull request by Jennycj**
The dockerfile builds but has errors when the container is run

### CODE REVIEW

**Comment by Jennycj** [comment]
> 

The error has been fixed.

File: src/loaders/express.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import express, { Request, Response, NextFunction } from "express";
 import cors from "cors";
-import routes from "@api/index";
-import config from "@config/index";
-import { CustomError, NotFoundError } from "@api/middlewares/error-handlers";
+import routes from "../api/index";
```
Comment (Extheoisah) [inline src/loaders/express.ts:3]: we want to preserve @ identifiers as it is easier to read and navigate. Pls revert this change

**Review by Extheoisah — CHANGES_REQUESTED** [review]
use @ identifiers

File: src/loaders/express.ts (hunk 1 above)
Comment (Jennycj) [reply src/loaders/express.ts:3]: That was the reason the app was not building. It wasn't able to resolve the paths. Do you have a workaround for that?

Comment (Extheoisah) [reply src/loaders/express.ts]: I have pushed a fix for it here -> https://github.com/Extheoisah/Mava/pull/8/commits/30072cdff76106708288cbf20494f7d1e6aeb0c7

I also fixed some issues with the model association.
